packages/mui-base/src/utils/visuallyHidden.ts

```diff
@@ -0,0 +1 @@
+export { default as visuallyHidden } from '@mui/utils/visuallyHidden';
```

Comment:
Is there a reason to re-export into the utils here? Can we import from `@mui/utils` directly?

Reply:
Given the [not-entirely-clear future of @mui/utils](https://www.notion.so/mui-org/mui-utils-purpose-9a9fc9da3a004864b6c4e1f4d1f24f95), I prefer to import these utilities locally, to reduce the number of changes in the future.